packages/libraries/cli/src/commands/app/create.ts

```diff
@@ -66,6 +72,15 @@ export default class AppCreate extends Command<typeof AppCreate> {
       throw new MissingRegistryTokenError();
     }
 
+    let target: GraphQLSchema.TargetSelectorInput | null = null;
+    if (flags.target) {
+      const result = TargetSlug.parse(flags.target);
+      if (result.type === 'error') {
+        throw new InvalidTargetError();
+      }
+      target = result.data;
+    }
```

Comment:
Not sure if already too aggressive: We could start logging a warning here.

Reply:
`process.exit(1)` to make sure people see it